src/main/java/org/fundacionjala/coding/dennis/BankOcr.java

```diff
@@ -0,0 +1,95 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
+ */
+public final class BankOcr {
+
+    private static final Map<String, String> PATTERN;
+    static {
+        Map<String, String> mapElem = new HashMap<>();
+        mapElem.put("     |  |", "1"); mapElem.put(" _  _||_ ", "2");
+        mapElem.put(" _  _| _|", "3"); mapElem.put("   |_|  |", "4");
+        mapElem.put(" _ |_  _|", "5"); mapElem.put(" _ |_ |_|", "6");
+        mapElem.put(" _   |  |", "7"); mapElem.put(" _ |_||_|", "8");
+        mapElem.put(" _ |_| _|", "9"); mapElem.put(" _ | ||_|", "0");
+        PATTERN = Collections.unmodifiableMap(mapElem);
+    }
+    public static final int STEP = 3;
+    public static final int MOD = 11;
+
+    /**
+     *
+     */
+    private BankOcr() {
+    }
+
+    /**
+     *
+     * @param s1 compenent 1.
+     * @param s2 component 2.
+     * @param s3 component 3.
+     * @return number decoded.
+     */
+    public static String decode(final String s1, final String s2, final String s3) {
+        //StringBuilder numPattern = new StringBuilder(1);
+        String numPatter = "";
+        String numResult = "";
+        for (int column = 0; column < s1.length(); column += STEP) {
+            String cad = numPatter.concat("")
+                    .concat(s1.substring(column, column + STEP))
+                    .concat(s2.substring(column, column + STEP))
+                    .concat(s3.substring(column, column + STEP));
+            //numPattern.setLength(0);
+            //numPattern.append(s1, column, column + STEP)
+            //        .append(s2, column, column + STEP)
+            //        .append(s3, column, column + STEP);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)